fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -156,18 +150,18 @@ RdfStream getTriples(final IdentifierConverter<Resource, FedoraResource> idTrans
                          final Set<? extends TripleCategory> contexts);
 
     /**
-     * Get the JCR Base version for the node
+     * Get the base version for the node
      *
      * @return base version
      */
-    public Version getBaseVersion();
+    FedoraResource getBaseVersion();
```

Comment:
I don't want to hold this PR up on this point, but for symmetry, it is a little odd that `getBaseVersion` returns a `FedoraResource` whereas `getVersions` returns `FedoraVersion`s.